src/DataAccess/Migrations/Version20240826150940.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace WMDE\Fundraising\MembershipContext\DataAccess\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240826150940 extends AbstractMigration {
+	public function getDescription(): string {
+		return '';
+	}
+
+	public function up( Schema $schema ): void {
+		$schema->getTable( 'request' )->getColumn( 'postcode' )->setLength( 16 );
```

Comment:
Sorry, the Migrations use the `real` column names from the database, not the ones from PHP
```suggestion
		$schema->getTable( 'request' )->getColumn( 'plz' )->setLength( 16 );
```